src/main/resources/i18n/messages.properties

```diff
@@ -2865,4 +2865,42 @@ FilterByFile.valid.single=One sample name has been matched to a project.
 FilterByFile.valid.plural={0} sample names have been matched to a project.
 FilterByFile.invalid=The following sample names could not be matched to a project.
 FilterByFile.filter=Filter
-FilterByFile.file-label=Select file containing sample names
\ No newline at end of file
+FilterByFile.file-label=Select file containing sample names
+
```

Comment:
You probably need to clean up some old ones right?

Reply:
Yup once I start fully cleaning it up